recipes/xxhash/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class XxHash(ConanFile):
+    name = "xxhash"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Cyan4973/xxHash"
+    description = "Extremely fast non-cryptographic hash algorithm"
+    topics = ("conan", "hash", "algorithm", "fast", "checksum", "hash-functions")
+    license = "BSD-2-Clause"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "xxHash-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake is None:
+            self._cmake = CMake(self)
+            self._cmake.definitions["XXHASH_BUNDLED_MODE"] = False
```

Comment:
```suggestion
            self._cmake.definitions["XXHASH_BUNDLED_MODE"] = False
            self._cmake.definitions["XXHASH_BUILD_XXHSUM"] = self.options.utility
```

Reply:
I am not sure about adding that option to build the executable. As it is very small I'd prefer to package it always and avoid adding an option for it, it is not very important

Reply:
Installation of executables through CMake is sometimes broken for iOS (because BUNDLE DESTINATION is not specified, and it's required since a CMake version I can't remember), I think that several consumers would appreciate this option to be able to disable build of utility.

Reply:
thanks for the helpful explanation. I will add you suggestions then 😄